var/spack/repos/builtin/packages/kahip/package.py

```diff
@@ -0,0 +1,82 @@
+##############################################################################
+# Copyright (c) 2017 Christian Schulz
```

Comment:
Just talked to @tgamblin and he is fine with people modifying the copyright header as long as it remains LGPL.